source/feature-manager.tsx

```diff
@@ -76,7 +76,7 @@ const logError = (url: string, error: unknown): void => {
 	newIssueUrl.searchParams.set('labels', 'bug');
 	newIssueUrl.searchParams.set('template', '1_bug_report.yml');
 	newIssueUrl.searchParams.set('title', `\`${id}\`: ${message}`);
-	newIssueUrl.searchParams.set('example_urls', location.href);
+	newIssueUrl.searchParams.set('repro', location.href);
 	newIssueUrl.searchParams.set('description', [
```

Comment:
#6300 should work now thanks to aed513d. I'm including it here because they're related